train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -80,7 +81,7 @@ GPT_ARGS=" \
     --micro-batch-size $MICRO_BATCH_SIZE \
     --rampup-batch-size 16 16 6_000_000 \
     --global-batch-size $GLOBAL_BATCH_SIZE \
-    --train-samples 300_000_000 \
+    --train-samples $((3000000000 / $SEQ_LEN + 1)) \
```

Comment:
This should be `1464844` right as all sequence are 2048 long? Also this formula becomes wrong with CL activate

Reply:
If I'm not mistaken you have mixed up `--train-tokens` here. This is samples, not tokens. 300M samples is ~600B tokens with seqlen 2048. So in the original it should have been `--train-samples 150_000_000` to match the 300B tokens, but it wasn't an issue as we stopped exactly at 150M samples.

Reply:
please let's not use this format, who can count all those zeros? it's too error-prone.

Reply:
I think the correct math is: `300_000_000_000/2048 = 146_484_375`

Reply:
I'd just set it at a round 150M